### DISCUSSION TITLE
2024-09-12

### DISCUSSION THREAD

**Discussion by pshriwise** [svalinn/DAGMC#961] [Community Calls]
XDG (DAGMC 2.0) links:

  - https://github.com/pshriwise/xdg (works with https://github.com/pshriwise/openmc/tree/xdg)
  - Memory usage improvement on double-down: https://github.com/pshriwise/xdg/issues/4
  - Embree conda package (for reference on selecting ISA config): https://github.com/conda-forge/embree-feedstock/blob/main/recipe/build.sh
  - New open source software conference for fusion energy: https://ossfe.github.io/

**Comment by gonuke** [comment]
@pshriwise I think your put the same link twice for XDG - both times it's the OpenMC branch...

**Reply by pshriwise** [reply]
Corrected. Thanks!
